### PR TITLE
Discord.js v11

### PR DESCRIPTION
**Pull request by jolewine**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,8 +26,9 @@
   },
   "homepage": "https://github.com/exoplay/exobot-adapter-discord#readme",
   "dependencies": {
-    "discord.js": "^9.3.0",
-    "@exoplay/exobot": ">=4.1 <5"
+    "@exoplay/exobot": ">=4.1 <5",
+    "discord.js": "^11.0.0",
```
Comment (ajacksified) [inline package.json:30]: The docs also suggest using `uws` - have you tried with that?

File: src/discord.js [hunk 2]
```diff
@@ -83,7 +96,7 @@ export default class DiscordAdapter extends Adapter {
 
     this.bot.emitter.emit('connected', this.name);
     this.bot.log.notice('Connected to Discord.');
-    this.client.user.game = 'Exobotting';
+    this.client.user.setGame('Exobotting');
```
Comment (ajacksified) [inline src/discord.js]: Configurable? I could see maybe wanting this to be a stream name or something. (I know it wasn't before, but may be an easy change to make here or in another PR.)

**Review by ajacksified — APPROVED** [review]
Approved, unless you wanna try out those two changes.

File: package.json (hunk 1 above)
Comment (jolewine) [reply package.json:30]: I was worried about the auto-reconnect breaking with uws.  When they put out the first patch for v11 I will revisit this.
